tests.foreman.sys.test_katello_certs_check.KatelloCertsCheckTestCase.test_positive_validate_katello_certs_check_output: /usr/sbin/katello-certs-check: illegal option -- r

```
Error Message

AssertionError: 1 != 0

Stacktrace

self = <tests.foreman.sys.test_katello_certs_check.KatelloCertsCheckTestCase testMethod=test_positive_validate_katello_certs_check_output>

    @tier1
    def test_positive_validate_katello_certs_check_output(self):
        """Validate that katello-certs-check generate correct output
    
            :id: 4c9e4c6e-8d8e-4953-87a1-09cb55df3adf
    
            :steps:
    
                1. Generate custom certs
                2. Run katello-certs-check with the required valid arguments
                   katello-certs-check -c CERT_FILE -k KEY_FILE -r REQ_FILE
                   -b CA_BUNDLE_FILE
                3. Assert the output has correct commands with options
    
            :expectedresults: Katello-certs-check should generate correct commands
             with options.
            """
        with get_connection() as connection:
            result = connection.run(
                'katello-certs-check -c /tmp/{0} -k /tmp/{1} -r /tmp/{2} '
                '-b /tmp/{3}'.format(
                    self.cert_file_name,
                    self.key_file_name,
                    self.req_file_name,
                    self.ca_bundle_file_name
                ),
                output_format='plain'
            )
>           self.validate_output(result)

tests/foreman/sys/test_katello_certs_check.py:119: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
tests/foreman/sys/test_katello_certs_check.py:71: in validate_output
    self.assertEqual(result.return_code, 0)
E   AssertionError: 1 != 0

Standard Output

2018-06-23 06:13:10 - robottelo - INFO - Started setUpClass: tests.foreman.sys.test_katello_certs_check/KatelloCertsCheckTestCase
2018-06-23 06:13:11 - robottelo.ssh - DEBUG - Instantiated Paramiko client 0x7f5d7e9ba630
2018-06-23 06:13:11 - robottelo.ssh - INFO - Connected to [None]
2018-06-23 06:13:11 - robottelo.ssh - DEBUG - Destroyed Paramiko client 0x7f5d7e9ba630
2018-06-23 06:13:11 - robottelo.ssh - DEBUG - Instantiated Paramiko client 0x7f5d7e554d30
2018-06-23 06:13:11 - robottelo.ssh - INFO - Connected to [None]
2018-06-23 06:13:11 - robottelo.ssh - DEBUG - Destroyed Paramiko client 0x7f5d7e554d30
2018-06-23 06:13:11 - robottelo.ssh - DEBUG - Instantiated Paramiko client 0x7f5d7e554d30
2018-06-23 06:13:11 - robottelo.ssh - INFO - Connected to [None]
2018-06-23 06:13:12 - robottelo.ssh - DEBUG - Destroyed Paramiko client 0x7f5d7e554d30
2018-06-23 06:13:12 - robottelo.ssh - DEBUG - Instantiated Paramiko client 0x7f5d7e9a2358
2018-06-23 06:13:12 - robottelo.ssh - INFO - Connected to [None]
2018-06-23 06:13:12 - robottelo.ssh - DEBUG - Destroyed Paramiko client 0x7f5d7e9a2358
2018-06-23 06:13:12 - robottelo - INFO - Entities cleaner disabled
2018-06-23 06:13:12 - robottelo - DEBUG - Started Test: KatelloCertsCheckTestCase/test_positive_validate_katello_certs_check_output
2018-06-23 06:13:12 - robottelo.ssh - DEBUG - Instantiated Paramiko client 0x7f5d7e9aba90
2018-06-23 06:13:12 - robottelo.ssh - INFO - Connected to [None]
2018-06-23 06:13:12 - robottelo.ssh - INFO - >>> katello-certs-check -c /tmp/server.valid.crt -k /tmp/server.key -r /tmp/server.csr -b /tmp/rootCA.pem
2018-06-23 06:13:14 - robottelo.ssh - INFO - <<< stderr
/usr/sbin/katello-certs-check: illegal option -- r
Verifies, that custom ssl cert files are usable
as part of the Katello installation.

usage: /usr/sbin/katello-certs-check -c CERT_FILE -k KEY_FILE -b CA_BUNDLE_FILE

2018-06-23 06:13:14 - robottelo.ssh - DEBUG - Destroyed Paramiko client 0x7f5d7e9aba90
2018-06-23 06:13:14 - robottelo - DEBUG - Finished Test: KatelloCertsCheckTestCase/test_positive_validate_katello_certs_check_output
2018-06-23 06:13:14 - robottelo - INFO - Started tearDownClass: tests.foreman.sys.test_katello_certs_check/KatelloCertsCheckTestCase
```

--- a/tests/foreman/sys/test_katello_certs_check.py
+++ b/tests/foreman/sys/test_katello_certs_check.py
@@ -37,11 +37,9 @@ class KatelloCertsCheckTestCase(TestCase):
         )
         _, cls.cert_file_name = os.path.split(settings.certs.cert_file)
         _, cls.key_file_name = os.path.split(settings.certs.key_file)
-        _, cls.req_file_name = os.path.split(settings.certs.req_file)
         cls.ca_bundle_file = settings.certs.ca_bundle_file
         cls.cert_file = settings.certs.cert_file
         cls.key_file = settings.certs.key_file
-        cls.req_file = settings.certs.req_file
         cls.SUCCESS_MSG = "Validation succeeded"
         # uploads certs to satellite
         upload_file(
@@ -56,16 +54,11 @@ class KatelloCertsCheckTestCase(TestCase):
             local_file=settings.certs.key_file,
             remote_file="/tmp/{0}".format(cls.key_file_name)
         )
-        upload_file(
-            local_file=settings.certs.req_file,
-            remote_file="/tmp/{0}".format(cls.req_file_name)
-        )
 
     def validate_output(self, result):
         expected_result = set(
             ['--certs-update-server-ca', '--certs-server-key', '--server-cert',
-                '--server-key', '--certs-server-cert', '--server-cert-req',
-                '--certs-server-cert-req', '--certs-update-server',
+                '--server-key', '--certs-server-cert', '--certs-update-server',
                 '--foreman-proxy-fqdn', '--scenario',
                 '--certs-tar', '--server-ca-cert', '--certs-server-ca-cert'])
         self.assertEqual(result.return_code, 0)
@@ -107,11 +100,10 @@ class KatelloCertsCheckTestCase(TestCase):
         """
         with get_connection() as connection:
             result = connection.run(
-                'katello-certs-check -c /tmp/{0} -k /tmp/{1} -r /tmp/{2} '
-                '-b /tmp/{3}'.format(
+                'katello-certs-check -c /tmp/{0} -k /tmp/{1} '
+                '-b /tmp/{2}'.format(
                     self.cert_file_name,
                     self.key_file_name,
-                    self.req_file_name,
                     self.ca_bundle_file_name
                 ),
                 output_format='plain'

